services/src/main/java/org/exoplatform/news/NewsTargetingServiceImpl.java

```diff
@@ -43,11 +46,21 @@ public NewsTargetingServiceImpl(MetadataService metadataService){
     List<Metadata> targets = metadataService.getMetadatas(METADATA_TYPE.getName(),  LIMIT);
     return targets.stream().map(this::toEntity).collect(Collectors.toList());
   }
+
+  @Override
+  public  List<NewsTargetingEntity> getReferencedTargets() {
```

Comment:
to be updated according to social [Social PR](https://github.com/Meeds-io/social/pull/1103)